test/e2e/performanceprofile/testdata/ppc-expected-profiles/profile1a.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
+    isolated: 1,3-39,41,43,45,47,49-51,53-79
+    reserved: 0,2,40,42
+    offlined: 44,46,48,52
```

Comment:
@jlojosnegros   in all the expected profile nder this test data , do we have the `offlined` always set with values ? 
If the answer is yes then I would add at least one test profile without that field (since its optional) to test the default behavior.